transport/local/transport.go

```diff
@@ -0,0 +1,64 @@
+package local
+
+import (
+	"io"
+	"os"
+	"sync"
+
+	"github.com/kcarretto/paragon/transport"
+)
+
+type Transport struct {
+	io.Reader
+	transport.PayloadWriter
+
+	wg     *sync.WaitGroup
+	input  chan []byte
+	closed bool
+}
+
+func (t Transport) Write(data []byte) (int, error) {
+	t.ensureActive()
+	return os.Stdout.Write(data)
+}
+
+func (t Transport) Close() error {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
exported method Transport.Close should have comment or be unexported